jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/data/ModelFeaturesConfig.scala

```diff
@@ -35,13 +35,14 @@ case class ModelFeaturesConfig(@JsonProperty("rank") initialRank: FeatureConfig,
 
 /**
   * At inference-time, ml4ir's FeatureConfig needs to know only the following properties per feature:
-  * @param nodeName the Tensorflow input graph node name to wire this feature into
+  * @param name the protobuf key name representing the input. Tensorflow will wire the value into the corresponding node
+  *             name
   * @param dTypeString string | int64 | float
   * @param servingConfig {@see ServingConfig} below
   * @param tfRecordType context | sequence (latter only for {@see SequenceExample} features)
   */
 @JsonIgnoreProperties(ignoreUnknown = true)
-case class FeatureConfig(@JsonProperty("node_name") nodeName: String,
+case class FeatureConfig(@JsonProperty("name") name: String,
```

Comment:
agreed. This raises the question if we need node_name and why. Same question for serving_info.name. 
Right now we have 3 names, I am wondering if we can live with 2.

Reply:
I think we need node_name (it has to be unique, and it is visible in the graph, so useful for debug to have meaningful name).
But maybe we could get rid of serving_info.name.

Reply:
BTW, I double-checked that 'name' is not used as a key to a map of feature info. In the python code, the features are always manipulated as a list. (so 'name' does not have to be unique)